simple/stats/db.py

```diff
@@ -92,6 +93,8 @@
 
 _INSERT_IMPORTS_STATEMENT = "insert into imports values(?, ?, ?)"
 
+_SELECT_TRIPLES_WITH_TYPE_OF = "select * from triples where subject_id in (select subject_id from triples where predicate = 'typeOf' and object_id = ?)"
```

Comment:
I am bit confused on what this query does?

Reply:
It first gets dcids of nodes whose `typeOf` is the specified type and then it gets all triples for those dcids.

For the SVG hierarchy case, we want all triples of all SVs and we make this query with `object_id = 'StatisticalVariable'` for that.

Reply:
I see, then maybe change the var name to _SELECT_TRIPLES_BY_SUBJECT_TYPE

Reply:
Done.